benchmark.py

```
@@ -240,7 +241,11 @@ async def insert_batch(session: aiohttp.ClientSession, url: str, batch: list[dic
     if not batch:
         return
     content = {'changes': batch}
-    async with session.post(url, json=content) as response:
+    headers = {
+        'Content-Type': 'application/vnd.msgpack',
+        'Accept': 'application/vnd.msgpack'
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)